### PR TITLE
Add crashlytics

### PR DESCRIPTION
**Pull request by TKBurner**
staging codelab docs today.

### CODE REVIEW

**Comment by TKBurner** [comment]
do not merge yet.

File: ios-starter/objc-starter/FriendlyChatObjC.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -116,16 +123,36 @@
...
 		1079607F1C6130EF003DB6CC /* FriendlyChatObjC */ = {
 			isa = PBXNativeTarget;
 			buildConfigurationList = 107960A21C6130EF003DB6CC /* Build configuration list for PBXNativeTarget "FriendlyChatObjC" */;
 			buildPhases = (
+				A9C34ECFB4DDCA6F9EAE2B6A /* [CP] Check Pods Manifest.lock */,
 				1079607C1C6130EF003DB6CC /* Sources */,
 				1079607D1C6130EF003DB6CC /* Frameworks */,
 				1079607E1C6130EF003DB6CC /* Resources */,
+				91C2F7EC4501A7E7ED3122EE /* [CP] Embed Pods Frameworks */,
+				12980CAF543D23BCBE2B26DF /* [CP] Copy Pods Resources */,
```
Comment (ulukaya) [inline ios-starter/objc-starter/FriendlyChatObjC.xcodeproj/project.pbxproj]: please run "pod deintegrate" to delete all references to cocoapods in .xcodeproj

File: ios/objc/FriendlyChatObjC/FCViewController.m [hunk 2]
```diff
@@ -16,6 +16,7 @@
 
 #import "Constants.h"
 #import "FCViewController.h"
+#import <Crashlytics/Crashlytics.h>
```
Comment (ulukaya) [inline ios/objc/FriendlyChatObjC/FCViewController.m]: would prefer new style `@import Crashlytics`

File: ios/objc/FriendlyChatObjC/Info.plist [hunk 3]
```diff
@@ -29,7 +29,7 @@
 			<string>Editor</string>
 			<key>CFBundleURLSchemes</key>
 			<array>
-				<string>REVERSED_CLIENT_ID</string>
+				<string>com.googleusercontent.apps.632453652117-8ed8g49mc2e1jlquobbnh9lcfnib8boe</string>
```
Comment (ulukaya) [inline ios/objc/FriendlyChatObjC/Info.plist]: revert this file

File: ios/swift/FriendlyChatSwift/Info.plist [hunk 4]
```diff
@@ -29,7 +29,7 @@
 			<string>Editor</string>
 			<key>CFBundleURLSchemes</key>
 			<array>
-				<string>REVERSED_CLIENT_ID</string>
+				<string>com.googleusercontent.apps.632453652117-8ed8g49mc2e1jlquobbnh9lcfnib8boe</string>
```
Comment (ulukaya) [inline ios/swift/FriendlyChatSwift/Info.plist]: revert pls

File: ios/swift/shared/Images.xcassets/AppIcon.appiconset/Contents.json [hunk 5]
```diff
@@ -97,6 +97,11 @@
       "idiom" : "ipad",
       "filename" : "logo1024-ipad-83@2x.png",
       "scale" : "2x"
+    },
+    {
+      "idiom" : "ios-marketing",
+      "size" : "1024x1024",
+      "scale" : "1x"
```
Comment (ulukaya) [inline ios/swift/shared/Images.xcassets/AppIcon.appiconset/Contents.json]: revert pls. I'll organize the image folder in a separate PR.

File: ios/swift/FriendlyChatSwift/Info.plist (hunk 4 above)
Comment (TKBurner) [reply ios/swift/FriendlyChatSwift/Info.plist]: done

File: ios/swift/shared/Images.xcassets/AppIcon.appiconset/Contents.json (hunk 5 above)
Comment (TKBurner) [reply ios/swift/shared/Images.xcassets/AppIcon.appiconset/Contents.json]: done

File: ios/objc/FriendlyChatObjC/FCViewController.m (hunk 2 above)
Comment (TKBurner) [reply ios/objc/FriendlyChatObjC/FCViewController.m]: done

File: ios/objc/FriendlyChatObjC/Info.plist (hunk 3 above)
Comment (TKBurner) [reply ios/objc/FriendlyChatObjC/Info.plist]: done

**Comment by TKBurner** [comment]
Thanks @ulukaya. Can you take another look?

File: ios/objc/FriendlyChatObjC.xcodeproj/project.pbxproj [hunk 6]
```diff
@@ -18,6 +18,7 @@
 		10AF4CC81DD237AD0096D192 /* Bladefile in Resources */ = {isa = PBXBuildFile; fileRef = 10AF4CC41DD237AD0096D192 /* Bladefile */; };
 		10AF4CC91DD237AD0096D192 /* Images.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 10AF4CC51DD237AD0096D192 /* Images.xcassets */; };
 		10AF4CCA1DD237AD0096D192 /* LaunchScreen.xib in Resources */ = {isa = PBXBuildFile; fileRef = 10AF4CC61DD237AD0096D192 /* LaunchScreen.xib */; };
+		778B6CD11FDF12FA004AF0AF /* GoogleService-Info.plist in Resources */ = {isa = PBXBuildFile; fileRef = 778B6CD01FDF12F9004AF0AF /* GoogleService-Info.plist */; };
```
Comment (ulukaya) [inline ios/objc/FriendlyChatObjC.xcodeproj/project.pbxproj]: please delete this reference

File: ios/objc/FriendlyChatObjC/FCViewController.m [hunk 7]
```diff
@@ -129,8 +129,9 @@ - (IBAction)didSendMessage:(UIButton *)sender {
 }
 
 - (IBAction)didPressCrash:(id)sender {
-  FIRCrashLog(@"Cause Crash button clicked");
-  assert(NO);
+    NSLog(@"Crash button pressed!");
+    [[Crashlytics sharedInstance] crash];
+    assert(NO);
 }
```
Comment (ulukaya) [inline ios/objc/FriendlyChatObjC/FCViewController.m]: nit indent 2 spaces

File: ios/swift/FriendlyChatSwift.xcodeproj/project.pbxproj [hunk 8]
```diff
@@ -187,11 +191,28 @@
...
+			inputPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "\"${PODS_ROOT}/Fabric/run\"";
+		};
+/* End PBXShellScriptBuildPhase section */
+
```
Comment (ulukaya) [inline ios/swift/FriendlyChatSwift.xcodeproj/project.pbxproj]: I believe this should be deleted as well. could be added as intruction if this script has to be added manually.

File: ios/swift/shared/Images.xcassets/AppIcon.appiconset/Contents.json [hunk 9]
```diff
@@ -103,4 +103,4 @@
     "version" : 1,
     "author" : "xcode"
   }
-}
\ No newline at end of file
+}
```
Comment (ulukaya) [inline ios/swift/shared/Images.xcassets/AppIcon.appiconset/Contents.json]: please revert this file.

File: ios/objc/FriendlyChatObjC/FCViewController.m (hunk 7 above)
Comment (TKBurner) [reply ios/objc/FriendlyChatObjC/FCViewController.m]: fixed

File: ios/objc/FriendlyChatObjC.xcodeproj/project.pbxproj (hunk 6 above)
Comment (TKBurner) [reply ios/objc/FriendlyChatObjC.xcodeproj/project.pbxproj]: fixed

File: ios/swift/FriendlyChatSwift.xcodeproj/project.pbxproj (hunk 8 above)
Comment (ulukaya) [reply ios/swift/FriendlyChatSwift.xcodeproj/project.pbxproj]: Yeah realized that script here is fine.

**Pull request merged**